wormhole-connect/src/config/mainnet.ts

```diff
@@ -104,6 +104,7 @@ export const MAINNET_TOKENS: { [key: string]: TokenConfig } = {
     decimals: 18,
     solDecimals: 8,
     wrappedAsset: 'WETH',
+    automaticRelaying: true,
```

Comment:
Actually, I don't think we need this on the token config. We already check isAcceptedToken on the relayer contract